cli/js/lib.deno.ns.d.ts

```diff
@@ -1668,11 +1668,12 @@ declare namespace Deno {
     Busy: ErrorConstructor;
   };
 
-  /** **UNSTABLE**: potentially want names to overlap more with browser.
```

Comment:
We don't. Even if appropriately named permissions existed on the web, it's nonsense to try and align ours so we can get away with putting Deno's permissions in `navigator`, since they will inevitably mean different things.

The strategy would be to leave `Deno.permissions` as is and _also_ have `navigator.permissions` to contain web-compatible permissions like "notifications" (#4257) and "persistent-storage" (#1657).